main.go

```diff
@@ -0,0 +1,186 @@
+		}
+
+		// Github
+		{
+			if string(cfg.GithubAPIToken) != "" {
+				if err := os.Setenv("DANGER_GITHUB_API_TOKEN", string(cfg.GithubAPIToken)); err != nil {
+					failf("Failed to set env DANGER_GITHUB_API_TOKEN, error: %s", err)
+				}
+			}
+
+			if string(cfg.GithubHost) != "" {
+				if err := os.Setenv("DANGER_GITHUB_HOST", string(cfg.GithubHost)); err != nil {
+					failf("Failed to set env DANGER_GITHUB_HOST, error: %s", err)
+				}
+			}
+
+			if string(cfg.GithubAPIBaseURL) != "" {
+				if err := os.Setenv("DANGER_GITHUB_API_BASE_URL", string(cfg.GithubAPIBaseURL)); err != nil {
+					failf("Failed to set env DANGER_GITHUB_API_BASE_URL, error: %s", err)
+				}
+			}
+		}
+
+		// Gitlab
+		{
+			if string(cfg.GitlabAPIToken) != "" {
+				if err := os.Setenv("DANGER_GITLAB_API_TOKEN", string(cfg.GitlabAPIToken)); err != nil {
+					failf("Failed to set env DANGER_GITLAB_API_TOKEN, error: %s", err)
+				}
+			}
+
+			if string(cfg.GitlabHost) != "" {
+				if err := os.Setenv("DANGER_GITLAB_HOST", string(cfg.GitlabHost)); err != nil {
+					failf("Failed to set env DANGER_GITLAB_HOST, error: %s", err)
+				}
+			}
+
+			if string(cfg.GitlabAPIBaseURL) != "" {
+				if err := os.Setenv("DANGER_GITLAB_API_BASE_URL", string(cfg.GitlabAPIBaseURL)); err != nil {
+					failf("Failed to set env DANGER_GITLAB_API_BASE_URL, error: %s", err)
+				}
+			}
+		}
+	}
+
+	//
+	// Check dependencies
+	log.Infof("Checking dependencies")
+	{
+		log.Printf("Bundler...")
+		if ok, err := bundlerInstalled(); err != nil {
+			failf("Failed to check bundler, error: %s", err)
+		} else if !ok {
+			log.Warnf(`Bundler is not installed`)
+			fmt.Println()
+			log.Printf("Install Bundler")
+
+			if cmds, err := installBundler(); err != nil {
+				failf("Failed to create Bundler install command, error: %s", err)
+			} else {
+				for _, cmd := range cmds {
+					if out, err := cmd.RunAndReturnTrimmedCombinedOutput(); err != nil {
+						if errorutil.IsExitStatusError(err) {
+							log.Warnf("%s failed: %s", out)
+						} else {
+							log.Warnf("%s failed: %s", err)
+						}
+						failf("Failed to install Bundler, error: %s", err)
+					}
+				}
+			}
+		}
+		log.Printf("Bundler installed")
+	}
+
+	// Danger
+	{
+		fmt.Println()
+		log.Infof("Installing dependencies from your gem file")
+
+		cmd := command.New("bundle", "install")
+		cmd.SetStdout(os.Stdout)
+		cmd.SetStderr(os.Stderr)
+		log.Printf("$ %s", cmd.PrintableCommandArgs())
+
+		if err := cmd.Run(); err != nil {
+			failf("Failed to run bundle install, error: %s", err)
+		}
+
+		fmt.Println()
+		log.Infof("Running danger")
+
+		cmd = command.New("bundle", "exec", "danger")
+		cmd.SetStdout(os.Stdout)
+		cmd.SetStderr(os.Stderr)
+		log.Printf("$ %s", cmd.PrintableCommandArgs())
+
+		if err := cmd.Run(); err != nil {
+			failf("Failed to run bundle exec danger, error: %s", err)
+		}
+	}
+
+	fmt.Println()
+	log.Donef("Done")
+}
+
+func bundlerInstalled() (bool, error) {
```

Comment:
I recommend inlining this functions body.

`if ok, err := rubycommand.IsGemInstalled("bundler", ""); err != nil { ` is just as expressive as `if ok, err := bundlerInstalled(); err != nil {`, but without an extra indirection. Also, `bundlerInstalled` does nothing else, apart from delegating the call.

You might consider introducing a const for bundler version instead of empty string though.